src/components/CFButton/CFButton.js

```diff
@@ -23,9 +24,9 @@ export class CFButton extends React.PureComponent {
       <img 
         src={this.getIcon(globalScore)}
         className={this.getClassNames()}
-        title="CaptainFact"
+        title={ this.props.t('CaptainFact') }
```

Comment:
perso j'aime bien avoir une assignation genre 
`const { t } = props` et utiliser `t` dans mon composant, je trouve ça un peu moins lourd

Reply:
Perso je fais un `const { t } = props` s'il y a plus de 2 / 3 utilisations dans la fonction, sinon `this.props.t()` comme ici